pydropsonde/helper/paths.py

```diff
@@ -153,21 +165,21 @@ def populate_sonde_instances(self, config) -> Dict:
         """Returns a dictionary of `Sonde` class instances for all A-files found in `flight_idpath`
         and also sets the dictionary as value of `Sondes` attribute
         """
-        afiles = self.get_all_afiles()
+        dfiles = self.get_all_dfiles()
 
         Sondes = {}
 
-        for a_file in afiles:
-            sonde_id = rr.get_sonde_id(a_file)
-            try:
+        for d_file in dfiles:
+            sonde_id = rr.get_sonde_id(d_file)
+            if pp(d_file.replace("D", "A")).is_file():
+                a_file = d_file.replace("D", "A")
```

Comment:
This is actually a good use case for the walrus operator:
```suggestion
            if pp(a_file := d_file.replace("D", "A")).is_file():
```